### PR TITLE
[WIP] Issue #2809 Apply anyuid scc only to myproject namespace

### LINKED ISSUE #2809: Webconsole Crashloop and Permission Issue
### General information

  * Minishift version: minishift v1.24.0+8a904d0 /minishift v1.23
  * OS: macOS
  * Hypervisor:  xhyve 

### Steps to reproduce

  1. minishift profile set demo ( with memory 8GB/ vcpus 4)
  2. minishift start 
  3. minishift console 
  4. 

### Expected

The minishift console should be opened in browser

### Actual

The browser opens the OpenShift console url but  with `HTTP 404`.  Doing `oc project openshift-web-console` , `oc get pods` 

```
NAME                          READY     STATUS             RESTARTS   AGE
webconsole-6df7dd6b7b-6msxj   0/1       CrashLoopBackOff   8          18m
```

### Logs

`oc logs <webconsole pod>`

```
W0917 15:45:19.252521       1 start.go:93] Warning: config.clusterInfo.loggingPublicURL: Invalid value: "": required to view aggregated container logs in the console, web console start will continue.
W0917 15:45:19.252616       1 start.go:93] Warning: config.clusterInfo.metricsPublicURL: Invalid value: "": required to view cluster metrics in the console, web console start will continue.
Error: unable to load server certificate: open /var/serving-cert/tls.crt: permission denied
Usage:
  origin-web-console [flags]
Flags:
      --alsologtostderr                                log to standard error as well as files
      --audit-log-format string                        Format of saved audits. "legacy" indicates 1-line text format for each event. "json" indicates structured json format. Requires the 'AdvancedAuditing' feature gate. Known formats are legacy,json. (default "json")
      --audit-log-maxage int                           The maximum number of days to retain old audit log files based on the timestamp encoded in their filename.
      --audit-log-maxbackup int                        The maximum number of old audit log files to retain.
      --audit-log-maxsize int                          The maximum size in megabytes of the audit log file before it gets rotated.
      --audit-log-path string                          If set, all requests coming to the apiserver will be logged to this file.  '-' means standard out.
      --audit-policy-file string                       Path to the file that defines the audit policy configuration. Requires the 'AdvancedAuditing' feature gate. With AdvancedAuditing, a profile is required to enable auditing.
      --audit-webhook-batch-buffer-size int            The size of the buffer to store events before batching and sending to the webhook. Only used in batch mode. (default 10000)
      --audit-webhook-batch-initial-backoff duration   The amount of time to wait before retrying the first failed requests. Only used in batch mode. (default 10s)
      --audit-webhook-batch-max-size int               The maximum size of a batch sent to the webhook. Only used in batch mode. (default 400)
      --audit-webhook-batch-max-wait duration          The amount of time to wait before force sending the batch that hadn't reached the max size. Only used in batch mode. (default 30s)
      --audit-webhook-batch-throttle-burst int         Maximum number of requests sent at the same moment if ThrottleQPS was not utilized before. Only used in batch mode. (default 15)
      --audit-webhook-batch-throttle-qps float32       Maximum average number of requests per second. Only used in batch mode. (default 10)
      --audit-webhook-config-file string               Path to a kubeconfig formatted file that defines the audit webhook configuration. Requires the 'AdvancedAuditing' feature gate.
      --audit-webhook-mode string                      Strategy for sending audit events. Blocking indicates sending events should block server responses. Batch causes the webhook to buffer and send events asynchronously. Known modes are batch,blocking. (default "batch")
      --config string                                  filename containing the WebConsoleConfig
      --contention-profiling                           Enable lock contention profiling, if profiling is enabled
      --enable-swagger-ui                              Enables swagger ui on the apiserver at /swagger-ui
      --log-flush-frequency duration                   Maximum number of seconds between log flushes (default 5s)
      --log_backtrace_at traceLocation                 when logging hits line file:N, emit a stack trace (default :0)
      --log_dir string                                 If non-empty, write log files in this directory
      --logtostderr                                    log to standard error instead of files (default true)
      --profiling                                      Enable profiling via web interface host:port/debug/pprof/ (default true)
      --stderrthreshold severity                       logs at or above this threshold go to stderr (default 2)
  -v, --v Level                                        log level for V logs
      --vmodule moduleSpec                             comma-separated list of pattern=N settings for file-filtered logging
F0917 15:45:19.253264       1 console.go:35] unable to load server certificate: open /var/serving-cert/tls.crt: permission denied
```

### PR DESCRIPTION
**Pull request by anjannath**
Fixes #2809

### CODE REVIEW

File: addons/anyuid/anyuid.addon [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # Name: anyuid
+# Var-Defaults: NAMESPACE=myproject
 # Description: Changes the default security context constraints to allow pods to run as any user.
 # Url: https://docs.okd.io/latest/creating_images/guidelines.html#openshift-origin-specific-guidelines
 
-oc adm policy add-scc-to-group anyuid system:authenticated
+oc adm policy add-scc-to-group anyuid system:authenticated -n #{NAMESPACE}
```
Comment (praveenkumar) [inline addons/anyuid/anyuid.addon]: @anjannath have you tried it after making this change? I mean does it now solve the issue for users?

Comment (anjannath) [reply addons/anyuid/anyuid.addon]: @praveenkumar Nope, it didn't solve the issue :(

**Comment by anjannath** [comment]
retest this please

**Comment by gbraad** [comment]
the issue persisted with this change? If so, either close or change state to WIP.

**Comment by anjannath** [comment]
Closing as the issue persists even after the changes!

**Pull request closed without merging**